Gemfile

```diff
@@ -6,3 +6,6 @@ gemspec
 gem 'pry'
 gem 'rspec'
 gem 'childprocess'
+gem 'bundler'
```

Comment:
Please change gemspec instead. On row where is a bundler change add_development_dependency to add_runtime_dependency.